internal/runner/rspec_test.go

```diff
@@ -152,3 +153,65 @@ func TestRspecDiscoveryPattern_ExcludePattern(t *testing.T) {
 		t.Errorf("Rspec.discoveryPattern() diff (-got +want):\n%s", diff)
 	}
 }
+
+func TestRspecGetExamples(t *testing.T) {
+	rspec := NewRspec("rspec")
+	files := []string{"fixtures/spec/spells/expelliarmus_spec.rb"}
+	got, err := rspec.GetExamples(files)
+
+	want := []plan.TestCase{
+		{
+			Format:     plan.TestCaseFormatExample,
+			Identifier: "./fixtures/spec/spells/expelliarmus_spec.rb[1:1]",
+			Name:       "disarms the opponent",
+			Path:       "./fixtures/spec/spells/expelliarmus_spec.rb:2",
+			Scope:      "Expelliarmus disarms the opponent",
+		},
+		{
+			Format:     plan.TestCaseFormatExample,
+			Identifier: "./fixtures/spec/spells/expelliarmus_spec.rb[1:2]",
+			Name:       "knocks the wand out of the opponents hand",
+			Path:       "./fixtures/spec/spells/expelliarmus_spec.rb:6",
+			Scope:      "Expelliarmus knocks the wand out of the opponents hand",
+		},
+	}
+
+	if err != nil {
+		t.Errorf("Rspec.GetExamples(%q) error = %v", files, err)
+	}
+
+	if diff := cmp.Diff(got, want); diff != "" {
+		t.Errorf("Rspec.GetExamples(%q) diff (-got +want):\n%s", files, diff)
+	}
+}
+
+func TestRspecGetExamples_WithOtherFormatters(t *testing.T) {
```

Comment:
Is it also worth adding a test for when a formatter with JSON is already specified. BKs `.rspec.ci` file will also add a JSON formatter with a different output location, so I'm interested in seeing how that will interact with ours: https://github.com/buildkite/buildkite/blob/0a0b32ac150fa1fcb5a92044d7a94e4c840a527c/.rspec.ci#L10